django/db/models/functions/datetime.py

```diff
@@ -223,6 +223,19 @@ def as_postgresql(self, compiler, connection, **extra_context):
             compiler, connection, template="STATEMENT_TIMESTAMP()", **extra_context
         )
 
+    def as_mysql(self, compiler, connection, **extra_context):
+        return self.as_sql(
+            compiler, connection, template="CURRENT_TIMESTAMP(6)", **extra_context
+        )
+
+    def as_sqlite(self, compiler, connection, **extra_context):
+        return self.as_sql(
+            compiler,
+            connection,
+            template="STRFTIME('%%%%Y-%%%%m-%%%%d %%%%H:%%%%M:%%%%f', 'NOW')",
```

Comment:
I moved arguments to the `extra_context` to avoid escaping `%`.

Reply:
Kind of a de-optimization... now this string formatting will be executed every time the database function is used, even though it has a single result.

Reply:
`%`-formatting is called anyway:
https://github.com/django/django/blob/f71b0cf769d9ac582ee3d1a8c33d73dad3a770da/django/db/models/expressions.py#L965
Do you think an extra value can cause a performance regression? :thinking: I was thinking about readability.

Reply:
Yeah the formatting is called, but it can skip creating a new string if there is no % in the template. This results in probably a 10's of nanoseconds difference:

```
In [24]: %timeit "1234" % {}
29.4 ns ± 0.0916 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)

In [25]: %timeit "%(x)s" % {"x": "1234"}
52.9 ns ± 0.139 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)
```

Not much in the grand scheme of things, so I don't really mind, but to me the readability cost is worth it.

Reply:
Reverted.